azure-pipelines/secure-development-tools.yml

```diff
@@ -9,3 +18,87 @@ steps:
   displayName: 'Run PoliCheck'
   inputs:
     targetType: F
+
+ ### Analyze managed and unmanaged binaries (exe, dll) for security vulnerabilities
+- task: BinSkim@3
+  displayName: Run BinSkim
+  inputs:
+    InputType: 'Basic'
+    Function: 'analyze'
+    AnalyzeTarget: '$(Build.SourcesDirectory)/bin/StreamJsonRpc/$(BuildConfiguration)/netstandard2.0/StreamJsonRpc.dll;$(Build.SourcesDirectory)/bin/StreamJsonRpc/$(BuildConfiguration)/netcoreapp2.1/StreamJsonRpc.dll'
```

Comment:
I want to avoid hard-coded paths if possible, and especially hidden hard-coded lists of our TFMs. For instance when we add a target for `net5.0`, we'll never think to add it to this list. Instead, we can find an existing script or add a new one to the `azure-pipelines/variables` directory that searches for and returns all the matching binaries, and then just use `$(ScriptName)` here.

Reply:
Done, I added a script in `azure-pipelines/variables`